### PR TITLE
Added icon to task and update the label for JDK Architecture

### PR DESCRIPTION
**Pull request by kelliejos**

### CODE REVIEW

File: Tasks/JavaToolInstaller/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 0,
         "Minor": 126,
```
Comment (jpricketMSFT) [inline Tasks/JavaToolInstaller/task.json]: I think you should use 127.0 since we are in sprint 127.

Comment (madhurig) [reply Tasks/JavaToolInstaller/task.json]: I think Kellie is planning to port to 126, so this is ok.

**Comment by stephenmichaelf** [comment]
@kelliejos Is this going in or can we close it?

**Comment by kelliejos** [comment]
@stephenmichaelf This can be closed. The changes were introduced in a different PR.

**Pull request closed without merging**